writer_test.go

```diff
@@ -84,6 +84,80 @@ func TestWriteArray(t *testing.T) {
 	}
 }
 
+func TestWriteIntMap(t *testing.T) {
+	testPatterns := []struct {
+		value map[int]interface{}
+		cbor  []byte
+	}{
+		{
+			map[int]interface{}{
+				1: "Zürich",
+				2: "Confédération Suisse",
+			},
+			[]byte{
+				0xA2, 0x01, 0x67, 0x5A, 0xC3, 0xBC, 0x72, 0x69, 0x63,
+				0x68, 0x02, 0x76, 0x43, 0x6F, 0x6E, 0x66, 0xC3, 0xA9,
+				0x64, 0xC3, 0xA9, 0x72, 0x61, 0x74, 0x69, 0x6F, 0x6E,
+				0x20, 0x53, 0x75, 0x69, 0x73, 0x73, 0x65},
+		},
```

Comment:
quick Q: where was this vector generated?

Reply:
I used the web based converter at: http://cbor.me/ to generate those bytes.